ygen/codegen.go

```diff
@@ -472,6 +392,52 @@ func (cg *YANGCodeGenerator) GenerateGoCode(yangFiles, includePaths []string) (*
 	}, nil
 }
 
+func generateEnumCode(goEnums map[string]*yangEnum) ([]string, string, util.Errors) {
```

Comment:
Sorry that I missed this before, but can we ensure that this function is commented?